test/integration/default/files/_mapdata/amazonlinux-2.yaml

```diff
@@ -28,6 +28,7 @@ values:
     cert.from.src.to.add: {}
     cert.to.add:
       cert: "-----BEGIN CERTIFICATE-----\n2MOCKED CERT\n-----END CERTIFICATE----- "
+  certs_update_cmd: null
```

Comment:
here too, are those `null` value absolutely needed, isn't the default value sufficient?

Reply:
This is the files needed to check what `map.jinja` load, it must reflect the content of `cert._mapdata`.